### ISSUE TITLE
exportToJson generate unusable export for table created with separate UNIQUE syntax.

### ISSUE THREAD

**Issue by twnaing** [capacitor-community/sqlite#561]
**Describe the bug**
`exportToJson('full')` create strange syntax when sqlite table is create with `UNIQUE` in `CREATE TABLE`

```sql
CREATE TABLE table1(
...
UNIQUE(column1, column2)
);
```
resulting json

```json
        {
          "column": "UNIQUE(column1§",
          "value": "column2)"
        }
```

**To Reproduce**

1. create migration with the UNIQUE syntax
2. call exportToJson
3. See error

**Expected behavior**
A synxtax that is valid for import.

**Desktop (please complete the following information):**

- OS: macOS
- Browser: chromium
- Version: 125.0.6422.60

**Additional context**

relevant part of package.json

```json
    "@capacitor-community/sqlite": "^5.7.2",
    "jeep-sqlite": "^2.7.1",
```

**Work around**

Use `CREATE UNIQUE INDEX table1_unique_index ON table1(column1, column2)` syntax instead

EDIT: add `UNIQUE` to create index

**Comment by jepiqueau** [comment]
@twnaing the way to do it till i made the fix is "CONSTRAINT CUN_column1_column2 UNIQUE (column1, column2)"

**Comment by jepiqueau** [comment]
@twnaing has been fixed in v5.7.3
